src/Winp/Form/ServiceForm.cs

```diff
@@ -46,7 +47,7 @@ public ServiceForm()
             configuration = new ApplicationConfig();
         }
 
-        var scheduler = TaskScheduler.FromCurrentSynchronizationContext();
+        _scheduler = TaskScheduler.FromCurrentSynchronizationContext();

```

Comment:
Previous approach was done on purpose to avoid accessing `_scheduler` before it gets initialized: using a local variable allows compiler to detect uninitialized accesses 🙂 No need to mix refactorings into this PR 🙂